proposals/lip-0051.md

```diff
@@ -205,41 +305,20 @@ supplyStoreSchema = {
 
 The default value for this substore is `{"totalSupply": 0}` serialized using `supplyStoreSchema`.
 
-#### Available Local ID Substore
-
-The Token module store contains an entry dedicated to storing information about the available local IDs:
-
-* The store prefix is set to `STORE_PREFIX_AVAILABLE_LOCAL_ID`.
-* Each store key is the empty bytes.
-* Each store value is the serialization of an object following `availableLocalIDStoreSchema`.
-
-```java
-availableLocalIDStoreSchema = {
-    "type": "object",
-    "required": ["nextAvailableLocalID"],
-    "properties": {
-        "nextAvailableLocalID": {
-            "dataType": "uint32",
-            "fieldNumber": 1
-        }
-    }
-}
-```
 
-The default value for this substore is `{"nextAvailableLocalID": 0}` serialized using `availableLocalIDStoreSchema`.
 
 #### Escrow Substore
 
 The Token module store contains an entry dedicated to storing information about native tokens which have been sent to another chain. The state contains an entry with:
 
-* The store prefix is set to `STORE_PREFIX_ESCROW`.
-* Each store key is the identifier of the chain to which the tokens are escrowed, and the local ID of the escrowed token: `uint32be(escrowedChainID)||uint32be(tokenLocalID)`.
+* The substore prefix is set to `SUBSTORE_PREFIX_ESCROW`.
+* Each store key is the identifier of the chain to which the tokens are escrowed, and the token ID of the escrowed token: `escrowedChainID + tokenID`.
```

Comment:
this should be just `tokenID`

Reply:
I think it is correct as is: escrow has as key both `chainID` and `tokenID`.

Reply:
ah yes, i think genesis schema `escrowChainID`  was removed, so i got confused. This is correct, but the genesis escrow schema should have `escrowChainID`?

Reply:
This is correct, added.